app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -50,6 +51,8 @@
     /**
      * Wikibase Identifier associated with media files
      */
+    @PrimaryKey
+    @NonNull
     private String pageId;
```

Comment:
I think pageId is frequently null? I had that Pr where we kept requesting `Mnull`

Reply:
It won't be null for Media. It can be null for other API calls where `MwQueryPage` is returned as response.

Reply:
Then `NonNull` is not very true if it can be null. Is the annotation a Room requirement on primary keys? I've forgotten